Api/HyAcademy.GraphQL/Query.cs

```diff
@@ -17,6 +17,10 @@ public string GetAuthValue([Service] IHttpContextAccessor contextAccessor)
 
     public Task<Course?> GetCourse([Service] IGetCourseQuery query, Guid id) => query.Execute(id);
 
+    public Task<IQueryable<Lesson>> GetLessons([Service] IGetLessonsQuery query) => query.Execute();
```

Comment:
```suggestion
```
Anh nghĩ là sẽ ko cần query này.

Khi add GraphQL query thì em phải nghĩ về chuyện UI sẽ query như thế nào. Cho lessons thì UI sẽ ko query 1 list of lesson như vầy bao giờ do là lesson nó nằm trong course. Khi đó có nghĩa là khi lấy 1 list lesson thì sẽ luôn lấy theo course.

Nếu nghĩ theo hướng đó thì sẽ thấy là cái `lessons` này nó cần courseId. Nhưng nếu change thành `lesson(courseId: UUID!): [Lesson!]!` thì cũng ko đúng lắm. Cách đúng nhất trong GraphQL là để người ta query trực tiếp từ course, nghĩa là cái root query này là ko cần thiết.
```
query GetLessons {
  course(id: ...) {
    id
    lessons {
      id
      ...
    }
  }
}
```